rest_framework_swagger/static/rest_framework_swagger/swagger-ui.js

```diff
@@ -1209,7 +1209,8 @@ ApiKeyAuthorization.prototype.apply = function (obj) {
     return true;
   } else if (this.type === 'header') {
     if(typeof obj.headers[this.name] === 'undefined') {
-      obj.headers[this.name] = this.value;
+      if (window.swaggerUi.api.securityDefinitions.api_key.prefix_token != null)
+        obj.headers[this.name] = window.swaggerUi.api.securityDefinitions.api_key.prefix_token + this.value;
```

Comment:
This will apply the prefix token to all items in the header. Even X-CSRFToken data gets prefixed which is incorrect.